tests/conversion

```diff
@@ -5,6 +5,7 @@ set -eu
 . ./bin/helpers
 
 install_deps jq unzip
+install_deps virt-v2v || true
```

Comment:
Should have added the comment. This package is not available on an older 20.04 runner. virt-v2v was part of libguestfs-utils previously

Reply:
ok, please can you tweak this so that we check if package is available before installing it, that way if package install fails for some other reason we wont skip the test further down and will fail instead.